src/features/Common/DeleteModal/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ModalDialog, ModalCloseButton } from '@edx/paragon';
+import { Button } from 'react-paragon-topaz';
+
+import 'features/Common/DeleteModal/index.scss';
+
+const DeleteModal = ({
+  isOpen, onClose, title, textModal, labelDeleteButton, handleDelete,
```

Comment:
Just a thought but this could look much better in this way:

```suggestion
  isOpen,
  onClose,
  title,
  textModal,
  labelDeleteButton,
  handleDelete,
```

What do u think?